package.json

```diff
@@ -85,7 +85,7 @@
     },
     {
       "path": "./packages/runtime/dist/browser/index.js",
-      "limit": "313B",
+      "limit": "507B",
```

Comment:
@madou: Gone up by `194B` 😞. See if you can find anything 🤔.

Reply:
i'll get it down to 450B 💪